### DISCUSSION TITLE
Just upgraded to Brackets 2.2.0 (Mac) and all my code is gone!

### DISCUSSION THREAD

**Discussion by Courseprofs** [brackets-cont/brackets#267] [General]
Upgraded to Brackets 2.2.0 on Mac M1, none of my code in the left pane is there any more, it looks like my files have disappeared. Any idea how I can retrieve these?

**Comment by abose** [comment]
The files are most likely not gone. Can you check in the folder again? It might have closed the open project

**Comment by Courseprofs** [comment]
It looks like by default Brackets saves it's files in the Application Folder under Brackets > Contents > Samples > Root >Getting Started which I had no idea about, I thought it was being saved locally to my default folder on my local disk - when you upgrade Brackets it seems that this Root folder got overwritten wiping out my code library. Perhaps it should be clearer about this to help others avoid this, now I know I'll make sure I save these on a removable disc or something.

**Reply by abose** [reply]
We have put in new safeguards for the exact same issue in phcode.dev to use documents folder for default project. so it should not be an issue in future versions based on phcode. Hope you have not lost any critical data.

**Comment by swiftsmail** [comment]
installed this on my mac (ventura) and it installs in apps and then keeps the app folder as my local folder and completely hoses the mac os. it won't run afterwards. what a major pain.
